native/src/components/__tests__/SearchListItem.spec.tsx

```diff
@@ -65,25 +79,55 @@ describe('SearchListItem', () => {
     assertHighlighting(getByText(contentWithoutHtml), false)
   })
 
-  it.skip('should show excerpt around query if only match in content', () => {
+  it('should show excerpt around query if only match in content', () => {
     const excerptBeforeQuery = 'This is a'
     const query = 'sample'
     const excerptAfterQuery = 'page'
 
     const { getByText } = renderSearchListItem(query)
 
-    assertHighlighting(getByText(category.title), false)
+    assertHighlighting(getByText(category.title), true) // Titles are bold all the time from HighlighterCategoryTitle
```

Comment:
💭 Maybe we should remove the assertions for the title being bold? It doesn't seem like something that needs to be tested